### DISCUSSION TITLE
Does quarto 1.3 require pandoc 3.0?

### DISCUSSION THREAD

**Discussion by moonpiedumplings** [quarto-dev/quarto-cli#5754] [Q&A] [answered]
### Description

I have quarto and pandoc installed using the [nix package manager](https://nixos.org). Nix only packages quarto 1.2, and pandoc 2.1.9. After tinkering with nix to get it to package quarto 1.3 for me, I get an error upon attempting to render my website: 

```
[nix-shell:~/vscode/quartotest]$ quarto render
[1/9] about.qmd
Could not find data file templates/styles.citations.html
```

I'm guessing this is because pandoc 2.1.9 does not come with this particular template, but I want to be sure before I go further, as I am finding getting pandoc 3.0 through nix to be a very frustrating process.

**Comment by cscheid** [comment, marked as answer]
Yes, it does require pandoc 3.0.
